web/api/version.thrift

```diff
@@ -0,0 +1,14 @@
+// -------------------------------------------------------------------------
+//  Part of the CodeChecker project, under the Apache License v2.0 with
+//  LLVM Exceptions. See LICENSE for license information.
+//  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// -------------------------------------------------------------------------
+
+namespace py Version_v6
+namespace js codeCheckerVersion_v6
+
+service codeCheckerVersionService {
```

Comment:
Should we introduce a bit more general service name? So it could be extended with other package related information if needed? Calling it version might be a bit to restrictive for later extensions if needed. @bruntib what do you think?

Reply:
Okay, as we discussed I changed it to `ServerInfo`.